<launch>
    <machine name="host" address="localhost" />

    <node machine="host" pkg="rc_image_w_tf" type="rc_line_detect.py" name="rc_line_detect">
        <param name="image" value="/usb_cam_node/image_raw" type="String"/>
	    <param name="gui" value="false" type="bool"/>
	    <param name="dim" value="1" type="int"/>
    </node>

    <node machine="host" pkg="rc_image_w_tf" type="image2steer.py" name="image2steer">
        <param name="model_name" value="model_only_normal" type="String"/>
	    <param name="testmode" value="false" type="bool"/>
    </node>
</launch>
